src/linear_algebra/bilinear_form.lean

```diff
@@ -948,39 +948,39 @@ end to_matrix
 
 end matrix
 
-namespace refl_bilin_form
-
-open refl_bilin_form bilin_form
+namespace bilin_form
 
 /-- The proposition that a bilinear form is reflexive -/
 def is_refl (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = 0 → B y x = 0
```

Comment:
Out of scope for this PR, but it would be good to mention these definitions in the module docstring at the top of the file.